idseq_dag/steps/generate_accession2taxid.py

```diff
@@ -78,26 +78,27 @@ def run(self):
                                 accession2taxid_db, taxid2wgs_accession_db,
                                 output_gz, output_wgs_gz)
 
+
     def output_dicts_to_db(self, mapping_files, wgs_accessions,
-                           accession2taxid_db, taxid2wgs_accession_db,
-                           output_gz, output_wgs_gz):
+                                accession2taxid_db, taxid2wgs_accession_db,
+                                output_gz, output_wgs_gz):
         # generate the accession2taxid db and file
-        accession_dict = IdSeqDict(accession2taxid_db, IdSeqDictValue.VALUE_TYPE_SCALAR)
-        batch_list = {}
-        with gzip.open(output_gz, "wt") as gzf:
-            for partition_list in mapping_files:
-                for partition in partition_list:
-                    with open(partition, 'r', encoding="utf-8") as pf:
-                        for line in pf:
-                            if len(line) <= 1:
-                                break
-                            fields = line.rstrip().split("\t")
-                            gzf.write(line)
-                            batch_list[fields[0]] = fields[2]
-                            if len(batch_list) >= BATCH_INSERT_SIZE:
-                                accession_dict.batch_inserts(batch_list.items())
-                                batch_list = {}
-            accession_dict.batch_inserts(batch_list.items())
+        with IdSeqDictForUpdate(accession2taxid_db, IdSeqDictValue.VALUE_TYPE_SCALAR) as accession_dict
```

Comment:
I believe this line is missing an end colon

Reply:
Good catch.  Thank you.